hyperspy/tests/signal/test_eds_tem.py

```diff
@@ -327,3 +327,48 @@ def test_manual_remove_element(self):
         assert (
             sorted(s._xray_markers.keys()) ==
             ['Zn_Ka', 'Zn_La'])
+
+
+@lazifyTestClass
```

Comment:
Agreed, I have done nothing to adapt the function for lazy signals. I was trying to work out a way to still allow Lazy_rebin by integer pixels from the original rebin function but I'm not sure if I succeeded with that...